### PR TITLE
Hide displaying tokens in homescreen behind a feature flag

### PR DESCRIPTION
**Pull request by isabellewei**
### Description

Using firebase feature flags, if `multiTokenShowHomeBalances` is true, the new multitoken UI is showed in the homescreen, otherwise the old `BalancesDisplay` is shown in the sidebar.

This is branched off of both #1294 and #1388 

### Tested

not tested

### How others should test

refer to #1388

### CODE REVIEW

File: packages/mobile/src/home/WalletHome.tsx [hunk 1]
```diff
@@ -46,6 +47,7 @@ interface StateProps {
   numberVerified: boolean
   cashInButtonExpEnabled: boolean
   balances: Balances
+  showTokensinHome: boolean
```
Comment (gnardini) [inline packages/mobile/src/home/WalletHome.tsx:50]: nit: typo `showTokensinHome` -> `showTokensInHome`

File: packages/mobile/src/home/WalletHome.tsx [hunk 2]
```diff
@@ -174,6 +177,13 @@ export class WalletHome extends React.Component<Props, State> {
       renderItem: () => <NotificationBox key={'NotificationBox'} />,
     })
 
+    if (this.props.showTokensinHome) {
```
Comment (gnardini) [inline packages/mobile/src/home/WalletHome.tsx:180]: I think this is repeated, we should remove the one right below this :)

**Review by gnardini — COMMENTED** [review]
Looks great! Just left a small issue, might have been a merge issue :)

File: packages/mobile/src/home/WalletHome.tsx (hunk 1 above)
Comment (isabellewei) [reply packages/mobile/src/home/WalletHome.tsx:50]: hm not sure what you mean here they are the same haha

Comment (gnardini) [reply packages/mobile/src/home/WalletHome.tsx:50]: It's the capitalization in the i showTokens **i** nHome vs showTokens **I** nHome

Comment (isabellewei) [reply packages/mobile/src/home/WalletHome.tsx:50]: oooo I see

**Comment by noisypigeon** [comment]
@gnardini @isabellewei seems there's still an issue with this PR.

**Review by gnardini — APPROVED** [review]

**Pull request merged**